src/test/resources/no-reflect-and-serialization-list

```diff
@@ -0,0 +1,91 @@
+org.kohsuke.github.AbuseLimitHandler
```

Comment:
I'm still not sure this list is correct, but issues here can be addressed as bug fixes.